spec/board_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require './lib/tic_tac_toe.rb'
+
+RSpec.describe Board do
+  describe ' #initialize_Board ' do
+    it ' test the correct number on each cell of the grid' do
+      board = Board.new
```

Comment:
#### `REQUIRED`

Following the above suggestion, this line is not needed anymore.